sci-electronics/systemc/systemc-2.3.0-r1.ebuild

```diff
@@ -45,7 +45,8 @@ src_prepare() {
 }
 
 src_configure() {
-	econf $(use_enable static-libs static) CXX=$(tc-getCXX)
```

Comment:
Why are you removing it? Are you sure that this does not change behavior? Note that some configure files divert from the default GNU behavior, in which case we *want* to force CXX. For example, some projects default to using clang when available.